static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -353,27 +369,139 @@ class RuleConditionsForm extends PureComponent<Props, State> {
             alertType={alertType}
             required
           />
-          <SelectControl
-            name="timeWindow"
-            styles={{
-              control: (provided: {[x: string]: string | number | boolean}) => ({
-                ...provided,
-                minWidth: 200,
-                maxWidth: 300,
-              }),
-              container: (provided: {[x: string]: string | number | boolean}) => ({
-                ...provided,
-                margin: `${space(0.5)}`,
-              }),
-            }}
-            options={this.timeWindowOptions}
-            required
-            isDisabled={disabled}
-            value={timeWindow}
-            onChange={({value}) => onTimeWindowChange(value)}
-            inline={false}
-            flexibleControlStateSize
-          />
+          {monitorType === MonitorType.CONTINUOUS && (
+            <SelectControl
+              name="timeWindow"
+              styles={{
+                control: (provided: {[x: string]: string | number | boolean}) => ({
+                  ...provided,
+                  minWidth: 200,
+                  maxWidth: 300,
+                }),
+                container: (provided: {[x: string]: string | number | boolean}) => ({
+                  ...provided,
+                  margin: `${space(0.5)}`,
+                }),
+              }}
+              options={this.timeWindowOptions}
+              required={monitorType === MonitorType.CONTINUOUS}
+              isDisabled={disabled}
+              value={timeWindow}
+              onChange={({value}) => onTimeWindowChange(value)}
+              inline={false}
+              flexibleControlStateSize
+            />
+          )}
+        </FormRow>
+      </Fragment>
+    );
+  }
+
+  renderMonitorTypeSelect() {
```

Comment:
😢 was considering it...

but rather than having 2 render patterns, I figured we should keep it consistent.

hopefully we'll eventually rip this all out anyways